peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
@ghaskins data stored in ledger is available to any replica on disk and could be retrieved by any operator of a replica bypassing any authorisation. 
Yes, the noop chain code should not be enabled by default.